modules/utils/filters.js

```diff
@@ -0,0 +1,16 @@
+// const sanitizeHtmlFunction = require('sanitize-html');
+// const textService = require('../core/server/services/text.server.service');
+
+export const limitTo = (text, length) => {
+  return text.substring(0, length);
+};
+
+export const plainTextLength = (text) => {
+  return text && typeof(text) === 'string' ? String(text).replace(/&nbsp;/g, ' ').replace(/<[^>]+>/gm, '').trim().length : 0; // eslint-disable-line angular/typecheck-string
+};
+
+export const sanitizeHtml = (text) => {
+  // TODO solve a problem with sanitize html
+  return text;
+  // return sanitizeHtmlFunction(text, textService.sanitizeOptions);
+};
```

Comment:
`sanitize-html` was causing errors and `text.server.service` shouldn't be imported from the backend files as @mrkvon noticed so it will be changed to some architecture with a general frontend backend config in this PR.